lib/devices/default.js

```diff
@@ -218,6 +236,229 @@ let defaults = {
+            Object.prototype.hasOwnProperty.call(valueObj.sys.available_updates, 'stable') &&
+            Object.prototype.hasOwnProperty.call(valueObj.sys.available_updates.stable, 'version')
+          ) {
+            return true;
+          }
+        }
+        return false;
+      }
+    },
+    common: {
+      'name': 'New firmware available',
+      'type': 'boolean',
+      'role': 'state',
+      'read': true,
+      'write': false
+    }
+  },
+  'firmwareupdate': {
+    mqtt: {
+      http_cmd: '/rpc/Shelly.Update',
+    },
+    common: {
+      'name': 'Update Firmware',
+      'type': 'boolean',
+      'role': 'button',
+      'read': false,
+      'write': true
+    }
+  },
+  'uptime': {
+    mqtt: {
+      http_publish: '/rpc/Shelly.GetStatus',
+      http_publish_funct: async (value) => { return value ? shellyHelper.uptimeString(JSON.parse(value).sys.uptime) : undefined; }
+    },
+    common: {
+      'name': 'Uptime',
+      'type': 'string',
+      'role': 'info',
+      'read': true,
+      'write': false
+    }
+  },
+  'version': {
+    mqtt: {
+      http_publish: '/rpc/Shelly.GetDeviceInfo',
+      http_publish_funct: async (value) => { return value ? JSON.parse(value).ver : undefined; }
+    },
+    common: {
+      'name': 'Firmware version',
+      'type': 'string',
+      'role': 'state',
+      'read': true,
+      'write': false
+    }
+  },
+  'hostname': {
+    mqtt: {
+      http_publish: '/rpc/Shelly.GetStatus',
+      http_publish_funct: async (value) => {
+        if (value) {
+          const valueObj = JSON.parse(value);
+
+          if (valueObj.wifi.sta_ip) {
+            return valueObj.wifi.sta_ip;
+          } else if (valueObj.eth.ip) {
+            return valueObj.eth.ip;
+          }
+        }
+
+        return undefined;
+      }
+    },
+    common: {
+      'name': 'Device Hostname',
+      'type': 'string',
+      'role': 'info.ip',
+      'read': true,
+      'write': false
+    }
+  },
+  'id': {
+    mqtt: {
+      http_publish: '/rpc/Shelly.GetDeviceInfo',
+      http_publish_funct: (value, self) => { return self.getDeviceId(); }
+    },
+    common: {
+      'name': 'Device Id',
+      'type': 'string',
+      'role': 'state',
+      'read': true,
+      'write': false
+    }
+  },
+  'type': {
+    mqtt: {
+      http_publish: '/rpc/Shelly.GetDeviceInfo',
+      http_publish_funct: (value, self) => { return self.getDeviceType(); }
+    },
+    common: {
+      'name': 'Device type',
+      'type': 'string',
+      'role': 'state',
+      'read': true,
+      'write': false
+    }
+  },
+  'model': {
+    mqtt: {
+      http_publish: '/rpc/Shelly.GetDeviceInfo',
+      http_publish_funct: (value, self) => { return value ? JSON.parse(value).model : undefined; }
+    },
+    common: {
+      'name': 'Authentication enabled',
+      'type': 'boolean',
+      'role': 'state',
+      'read': true,
+      'write': false
+    }
```

Comment:
Ist mir im Log aufgefallen, als der Adapter nach dem Update neu gestartet: `
State value to set for "shelly.0.shellypro4pm#xxx#1.model" has to be type "boolean" but received type "string"` und konnte es jetzt auf diese Zeilen zurückführen. Das `authEnabled` ist ins `model` gerutscht und deshalb auch in der Baumansicht der Objekte als solches bezeichnet.